### DISCUSSION TITLE
BATTERY SETTING

### DISCUSSION THREAD

**Discussion by Viper3000** [seerge/g-helper#1980] [General]
![25  1](https://github.com/seerge/g-helper/assets/144729297/6622656b-f169-4f0f-a0bb-ad72cb041757)

When I set the battery in MyAsus and also in G-Helper to 60%, the battery still charges up to 100% .... Why?

**Comment by seerge** [comment]
@Viper3000 on the screenshot it's not charging right now

Keep in mind - after each restart, BIOS resets all settings to defaults, including charge limit. As soon as you log in and G-Helper starts it would apply charge limit again. Before app starting - your laptop may keep charging. This is normal and this is how things are.

https://github.com/seerge/g-helper/wiki/FAQ#battery-charge-limiter-is-not-working

**Reply by Viper3000** [reply]
Thank you for for a quick answer  and thank you also for the great work on the G-Helper program
